app/controllers/api_controller.rb

```diff
@@ -223,7 +223,25 @@ def validate_include_params
   end
 
   def included_relationships
-    IncludeParamHandler.new(params).call
+    @included_relationships ||= IncludeParamHandler.new(params).call
+  end
+
+  def included_db_relationships
+    @included_db_relationships ||= included_relationships.map { |value| cast_relationship_to_symbol_hash(value) } if included_relationships.present?
+  end
+
+  def cast_relationship_to_symbol_hash(value)
+    parts = value.split('.', 2)
+    if parts.length == 1
+      # special hack to allow for the flags table renamed in the backend but not in the FE includes
+      if parts.first == 'flags'
+        :framework_flags
```

Comment:
Ew. It's unfortunate that the naming diverged between the API and the model, this has caused lots of hacky workarounds for the serialization stuff. I wonder if in a future PR we could look at renaming the model instead @cwrw ?

Reply:
There is a bit of misunderstanding to what this is. This isn't a rename issue, but rather how we have chosen to surface PER relationships in the API. We don't prefix any relationship with `framework_`, this includes `responses` as well as `questions`

Reply:
also I wouldn't say it's hacky in serialization, these were decisions we took at the time before mapping exact includes received in requests came in. Serializers provide support for having custom names for relationships, and we've utilised those correctly.

Reply:
I've replaced this with a `db_alias` method in `app/services/include_param_handler.rb`